README.md

```diff
@@ -1,7 +1,61 @@
-This repository contains the latest code developed by Gluon for the MyDevoxx client application. This code itself builds upon earlier conference implementations developed by Gluon, and makes use of Gluon Mobile for cross-platform iOS and Android support. Functionality includes session summary, presenter list, as well as a number of other views related to venues, etc. Source code is released as a sample of a professional Gluon Mobile application and therefore makes use of best practices.
+# Devoxx
 
-If you have any questions, review support options at http://www.gluonhq.com/support.
+_Devoxx app for Android and iOS mobile devices_
 
-Gluon offers custom consulting services to develop cross-platform mobile applications. If you want to learn more, our services are detailed at http://www.gluonhq.com/services.
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+
+[Devoxx](https://beta.devoxx.com/) is a set of international conferences for developers, by developers.
```

Comment:
> from developers, for developers

That's the subtitle on the website that is referenced at the beginning of the sentence.